span.go

```diff
@@ -90,39 +107,20 @@ type Span interface {
 
 	// Provides access to the Tracer that created this Span.
 	Tracer() Tracer
+
+	// DEPRECATED
```

Comment:
SGTM. Just adding a reference to the not deprecated alternative should be good enough then.